test/applications_unit_test.go

```diff
@@ -252,11 +223,6 @@ func weMakeALookupApplicationsCall(applicationID int) error {
 
 func ApplicationsUnitContext(s *godog.Suite) {
 	// @unit.transactions
-	s.Step(`^a signing account with address "([^"]*)" and mnemonic "([^"]*)"$`, aSigningAccountWithAddressAndMnemonic)
-	s.Step(`^I build an application transaction with operation "([^"]*)", application-id (\d+), sender "([^"]*)", approval-program "([^"]*)", clear-program "([^"]*)", global-bytes (\d+), global-ints (\d+), local-bytes (\d+), local-ints (\d+), app-args "([^"]*)", foreign-apps "([^"]*)", foreign-assets "([^"]*)", app-accounts "([^"]*)", fee (\d+), first-valid (\d+), last-valid (\d+), genesis-hash "([^"]*)", extra-pages (\d+)$`, iBuildAnApplicationTransactionUnit)
-	s.Step(`^sign the transaction$`, signTheTransaction)
-	s.Step(`^the base(\d+) encoded signed transaction should equal "([^"]*)"$`, theBaseEncodedSignedTransactionShouldEqual)
```

Comment:
The functions `theBaseEncodedSignedTransactionShouldEqual` and `iBuildAnApplicationTransactionUnit` are no longer needed (they actually became `theBase64EncodedSignedTransactionShouldEqual` and `buildLegacyAppCallTransaction` in `test/transactions_test.go`, so can you remove them from this file as well?